tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreSampleAppsTest.cs

```diff
@@ -28,9 +28,36 @@ public DotNetCoreSampleAppsTest(ITestOutputHelper output) : base(output)
         private DockerVolume CreateSampleAppVolume(string sampleAppName) =>
             DockerVolume.CreateMirror(Path.Combine(_hostSamplesDir, "DotNetCore", sampleAppName));
 
+        [Fact, Trait("category", "latest")]
+        public void PipelineTestInvocationLatest()
+        {
+            Builds_NetCore21App_UsingNetCore21_DotNetSdkVersion(Settings.BuildImageName);
+            GDIPlusLibrary_IsPresentInTheImage("latest");
+        }
+
+        [Fact, Trait("category", "ltsversions")]
+        public void PipelineTestInvocationLtsVersions()
+        {
+            Builds_NetCore21App_UsingNetCore21_DotNetSdkVersion(Settings.LtsVersionsBuildImageName);
+            GDIPlusLibrary_IsPresentInTheImage("lts-versions");
+        }
+
+        [Fact, Trait("category", "vso-focal")]
+        public void PipelineTestInvocationVsoFocal()
+        {
+            GDIPlusLibrary_IsPresentInTheImage("vso-focal");
+        }
+
+        [Fact, Trait("category", "githubactions")]
+        public void PipelineTestInvocation()
+        {
+            GDIPlusLibrary_IsPresentInTheImage("github-actions");
+            GDIPlusLibrary_IsPresentInTheImage("github-actions-buster");
+        }
```

Comment:
Side note:
Xunit has a limitation with invoking a specific `category:parameter` so we went with this approach of wrapping a functions with a specific `category:parameter`. The feature to filter by `category:parameter` expected to be released in `v3` of xunit, Per requests back from 2020.
https://github.com/xunit/xunit/issues/2177#issuecomment-719983488
https://github.com/xunit/xunit/issues/1758#issuecomment-644272196
https://github.com/xunit/xunit/issues/2133